client/my-sites/marketing/paths.ts

```diff
@@ -0,0 +1,7 @@
+export const marketingTraffic = (siteSlug?: string | null ): string => {
+	return `/marketing/traffic/${ siteSlug || '' }`;
+};
+
+export const marketingSharingButtons = (siteSlug?: string | null ): string => {
+	return `/marketing/sharing-buttons/${ siteSlug || '' }`;
+};
```

Comment:
fun idea!

```ts
type SiteSlug = string;
type CalypsoPath = string;

export const marketingTraffic = (siteSlug?: SiteSlug | null ): CalypsoPath => {
	return `/marketing/traffic/${ siteSlug || '' }`;
};

export const marketingSharingButtons = (siteSlug?: SiteSlug | null ): CalypsoPath => {
	return `/marketing/sharing-buttons/${ siteSlug || '' }`;
};
```

these type aliases might convey more meaningful information at the call site. I find that useful. with enough uses of a given alias we can think about extracting it higher up. `getSelectedSiteSlug()` and friends should return these types too… `SiteSlug | null`

<img width="963" alt="Screen Shot 2019-04-30 at 5 51 25 PM" src="https://user-images.githubusercontent.com/5431237/56998374-ab8a7d80-6b70-11e9-8813-d9ad34e43f8c.png">